neurovault/apps/statmaps/templates/statmaps/gene_expression.html.haml

```diff
@@ -19,7 +19,7 @@
         			"columnDefs": [
 										{
 											"render": function ( data, type, row ) {
-												return "<a href='http://www.ncbi.nlm.nih.gov/gene/" + row[1] +"'>" + data + "</a>";
+												return data + " <a href='http://www.ncbi.nlm.nih.gov/gene/" + row[1] +"'>[NCBI]</a> <a href='http://neurosynth.org/genes/" + data +"'>[Neurosynth]</a>";
```

Comment:
No, it's just displayed before the links. So it looks like:

PPTC7 [[NCBI](http://www.ncbi.nlm.nih.gov/gene/160760)] [[Neurosynth](http://neurosynth.org/genes/PPTC7)]